funcx_endpoint/tests/test_rabbit_mq/test_endpoint.py

```diff
@@ -0,0 +1,124 @@
+import json
+import logging
+import multiprocessing
+import random
+import uuid
+
+import pika
+
+from funcx_endpoint.endpoint.rabbit_mq import (
+    ResultQueuePublisher,
+    ResultQueueSubscriber,
+    TaskQueuePublisher,
+    TaskQueueSubscriber,
+)
+
+LOG_FORMAT = "%(levelname) -10s %(asctime)s %(name) -20s %(lineno) -5d: %(message)s"
+logging.basicConfig(level=logging.INFO, format=LOG_FORMAT)
+logger = logging.getLogger(__name__)
+
+ENDPOINT_ID = "231fab4e-630a-4d76-bbbb-cf0b4aedbdf9"
+CONN_PARAMS = pika.URLParameters("amqp://guest:guest@localhost:5672/")
```

Comment:
Each test module seems to have this same parameter declaration, which is going to pose a problem if anyone ever has rabbit running on another port, or when credentialed testing begins.

Some of this will probably need tinkering then anyway, but I'd consider putting this into a fixture in the conftest, and then depending on it throughout the tests. It removes some duplication/repetition and is probably closer to what we'll ultimately need.